src/python/bot/tasks/fuzz_task.py

```diff
@@ -1242,6 +1243,55 @@ def key_fn(crash):
   return new_crash_count, known_crash_count, processed_groups
 
 
+def setup_auxiliary_build():
+  """Sets up auxiliary builds when necessary (e.g. DFSan build)."""
```

Comment:
please see comment in build_manager.py re refactoring and using RegularBuild instead.

Reply:
Ack, replied!

Reply:
Ok, done! Didn't get a chance to do a thorough self-review today, so the PR might not be production ready, but PTAL if you get a chance :)